website/build.gradle

```diff
@@ -104,3 +106,32 @@ task testWebsite(type: Exec) {
 task preCommit {
   dependsOn testWebsite
 }
+
+task mergeWebsite << {
```

Comment:
Here's the script used by another Apache project to manage the same process: https://github.com/apache/guacamole-website/blob/22043e1c01a1808a3c95b87a901fee45737259ed/build.sh#L136

There are a few additional steps which they perform which I believe would make sense to include:

- Move generated contents to a temp directory outside the enlistment. This would ensure we don't hit any issues when switches branches.
- Nuke previous contents from `contentRepoDir` before copying new contents. This ensures that any removed files also get removed from the branch.

Reply:
The org.ajoberstar.git-publish handles moving the files now, and creates a new git repository with the changes.